component-catalog/src/Examples/Block.elm

```diff
@@ -406,6 +411,104 @@ example =
                                 ]
                             ]
                   }
+                , { pattern =
+                        Code.fromModule moduleName "view "
+                            ++ Code.listMultiline
+                                [ Code.fromModule moduleName "emphasize"
+                                , Code.fromModule moduleName "content "
+                                    ++ Code.listMultiline
+                                        [ "…"
+                                        , Code.fromModule moduleName "bold ("
+                                            ++ (Code.fromModule moduleName "phrase " ++ "\"[bold text]\"")
+                                            ++ ")"
+                                        , Code.fromModule moduleName "italic ("
+                                            ++ (Code.fromModule moduleName "phrase " ++ "\"[italic text]\"")
+                                            ++ ")"
+                                        , "…"
+                                        ]
+                                        2
+                                ]
+                                1
+                  , description = "**Bold & Italic markdown**\n\nItalicize titles and highlight specific words inside of an emphasis block."
+                  , example =
+                        inParagraph
+                            [ Block.view [ Block.plaintext "The " ]
+                            , Block.view
+                                [ Block.emphasize
+                                , (List.concat >> Block.content)
+                                    [ [ Block.bold (Block.phrase "quick ") ]
+                                    , [ Block.italic (Block.phrase "brown ") ]
+                                    , Block.phrase "fox"
+                                    ]
+                                ]
+                            , Block.view [ Block.plaintext " jumped over the lazy dog." ]
+                            ]
+                  }
+                ]
+            , Heading.h2
+                [ Heading.plaintext "Blank width guidance"
+                , Heading.css [ Css.marginTop Spacing.verticalSpacerPx ]
+                ]
+            , Text.smallBody [ Text.markdown "The accessible name of all blanks, regardless of character width used, is \"blank.\"" ]
+            , Text.smallBody [ Text.markdown "If we're looking for a specific length of content to put in the blank, that _must_ be communicated elsewhere on the page to provide an equitable experience." ]
+            , Text.smallBody [ Text.markdown "The `characterWidth` parameter uses the number of characters expected in the blank to calculate a rough monospace-based width that visually looks like it _could_ match." ]
```

Comment:
I LOVE having this content moved up above the table! Such a great change -- way more readable, and totally fixes a _very_ funky use of the Table 😅